src/controllers/authenticationController.ts

```diff
@@ -115,6 +122,12 @@ export default class AuthenticationController {
     //
     // decode the SPID assertion into a SPID user
     //
```

Comment:
This comments refers to `const errorOrSpidUser = validateSpidUser(userPayload);`. Move it down

Reply:
done in https://github.com/pagopa/io-backend/pull/1012/commits/3ada102f061d73a9a4fc9258f840af9f6d3e6b7b